chpl/chpl-api/src/main/java/gov/healthit/chpl/CHPLConfig.java

```diff
@@ -66,7 +68,7 @@ public CommonsMultipartResolver getResolver() throws IOException {
         CommonsMultipartResolver resolver = new CommonsMultipartResolver();
 
         // Set the maximum allowed size (in bytes) for each individual file.
-        resolver.setMaxUploadSize(5242880);// 5MB
+        resolver.setMaxUploadSize(MAX_UPLOAD_FILE_SIZE); // 5MB
```

Comment:
Suggest moving `//5MB` to the declaration above, and adding something for the cookie age (even if it's just the type of number [seconds vs minutes...])